addOns/exim/src/main/java/org/zaproxy/addon/exim/PopupMenuCopyUrls.java

```diff
@@ -48,6 +50,7 @@ protected void performHistoryReferenceActions(List<HistoryReference> hrefs) {
         for (HistoryReference href : hrefs) {
             sb.append(href.getURI().toString());
             sb.append('\n');
+            Stats.incCounter(ExtensionExim.STATS_PREFIX + STATS_COPY_URL);
```

Comment:
Could concatenate in the constant (same in other places).

Reply:
Do you mean just use the strings directly instead of declaring constant?

Reply:
Meant `private static final String STATS_COPY_URL = ExtensionExim.STATS_PREFIX + "copy.url";`

Reply:
Oh okay. Got ya.